adserver/views.py

```diff
@@ -768,6 +774,43 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class PublisherPlacementReportView(PublisherAccessMixin, BaseReportView):
```

Comment:
I do wonder if we even need this report at all. The only difference is the ability to filter by div. Couldn't we just only show the Divs dropdown on the regular publisher report if the publisher has the flag turned on?

Reply:
Yea, that's how I had it originally, but since it's using a different table, "All placements" will show a different report than "All views" because we aren't always recording a placement. We could have one report with a default of None, and then "all" be a different view, but it feels weird.

Reply:
I guess also we can't backfill it.